test/integration/scheduler/preemption_test.go

```diff
@@ -1307,137 +1307,3 @@ func TestPDBInPreemption(t *testing.T) {
 		})
 	}
 }
-
-func initTestPreferNominatedNode(t *testing.T, nsPrefix string, opts ...scheduler.Option) *testutils.TestContext {
-	testCtx := testutils.InitTestSchedulerWithOptions(t, testutils.InitTestAPIServer(t, nsPrefix, nil), opts...)
-	testutils.SyncInformerFactory(testCtx)
-	// wraps the NextPod() method to make it appear the preemption has been done already and the nominated node has been set.
-	f := testCtx.Scheduler.NextPod
-	testCtx.Scheduler.NextPod = func() (podInfo *framework.QueuedPodInfo) {
-		podInfo = f()
-		podInfo.Pod.Status.NominatedNodeName = "node-1"
-		return podInfo
-	}
-	go testCtx.Scheduler.Run(testCtx.Ctx)
-	return testCtx
-}
-
-// TestPreferNominatedNode test when the feature of "PreferNominatedNode" is enabled, the overall scheduling logic is not changed.
-// If the nominated node pass all the filters, then preemptor pod will run on the nominated node, otherwise, it will be scheduled
-// to another node in the cluster that ables to pass all the filters.
-// NOTE: This integration test is not intending to check the logic of preemption, but rather a sanity check when the feature is
-// enabled.
-func TestPreferNominatedNode(t *testing.T) {
-	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.PreferNominatedNode, true)()
```

Comment:
why remove this test?

Reply:
with the feature gate removed, I was thinking this general case should be covered somewhere already. I went through all those test in the file, seems this test is an additional one, will add it back.